src/main/scala/cromwell/engine/backend/jes/JesEngineFunctions.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.engine.backend.jes
+
+import cromwell.binding.values._
+import cromwell.engine.EngineFunctions
+import cromwell.util.GcsUtil
+
+import scala.util.{Success, Try}
+
+/**
+ * Implementation of engine functions for the JES backend.
+ */
+class JesEngineFunctions(secretsFile: String) extends EngineFunctions {
```

Comment:
Is your stringly typed `secretsFile` really a `File` (in which case prefer using the newer `Path`) or is this so it can also be a GCS path?